src/js/views/patients/sidebar/action/action-sidebar-activity-views.js

```diff
@@ -75,6 +60,22 @@ const ActivityView = View.extend({
 
     return Templates[this.model.get('type')];
   },
+  templateContext() {
+    const editor = this.model.getEditor();
+    const clinician = this.model.getClinician();
+    const to_clinician = this.model.getClinician() ? `${ clinician.get('first_name') } ${ clinician.get('last_name') }` : null;
```

Comment:
I'm surprised this didn't fail the linter.. while we can send snake cased to templates, locally they should be camel cased and translated at the object `to_clinician: toClinician`

Reply:
should also probably be `clinician ?`  However it looks to me like it will always be true?  `getClinician` `Radio.request('entities', 'clinicians:model', this.get('_clinician'));` will always return at least an empty clinician model

Reply:
you could also just always do this instead of `: null` and `_.trim` it which for empty clinician models will result in an empty string which'd be nullish